webpack.config.js

```diff
@@ -1,7 +1,22 @@
 var path = require('path')
 const CopyPlugin = require('copy-webpack-plugin')
+const fs = require('fs')
+const SvgoInstance = require('svgo')
+
 const entry = require('./package.json').main
 
+const svgo = new SvgoInstance()
+
+let iconName
+try {
+  iconName = JSON.parse(fs.readFileSync('manifest.konnector', 'utf8')).icon
+  // we run optimize only on SVG
+  if (!iconName.match(/\.svg$/)) iconName = null
+} catch (e) {
+  // console.error(`Unable to read the icon path from manifest: ${e}`)
+}
+const appIconRX = iconName && new RegExp(`[^/]*/${iconName}`)
```

Comment:
Wouldn't it be more logical to move the regex to l41?
l41 would become
if(iconName && path.match(new RegExp([^/]*/${iconName})))
and it will remove the need to have appIconRX const

Reply:
You are actually right. This comes from the template and I did just copy it. This is fixed now.